src/extensions/step_handlers/swupdate_handler/src/swupdate_handler.cpp

```diff
@@ -278,38 +290,50 @@ ADUC_Result SWUpdateHandlerImpl::Install(const tagADUC_WorkflowData* workflowDat
 ADUC_Result SWUpdateHandlerImpl::Apply(const tagADUC_WorkflowData* workflowData)
 {
     ADUC_Result result = { ADUC_Result_Failure };
-    char* workFolder = NULL;
+    const ADUC_ConfigInfo* config = nullptr;
+    char* workFolder = nullptr;
 
     if (workflow_is_cancel_requested(workflowData->WorkflowHandle))
     {
         return this->Cancel(workflowData);
     }
 
+    config = ADUC_ConfigInfo_GetInstance();
+    if (config == nullptr)
+    {
+        Log_Error("Failed to get config info instance.");
```

Comment:
Probably do not need log as this should never really be NULL in practice

Reply:
Removed